ktor-client/ktor-client-features/ktor-client-json/common/src/io/ktor/client/features/json/JsonFeature.kt

```diff
@@ -59,17 +59,33 @@ class JsonFeature internal constructor(
          */
         var serializer: JsonSerializer? = null
 
+        /**
+         * Backing field with mutable list of content types that are handled by this feature.
+         */
+        private val acceptContentTypesMutable: MutableList<ContentType> = mutableListOf(ContentType.Application.Json)
```

Comment:
We have the convention of naming mutable backing fields as: `_ acceptContentTypes`